src/kudu/consensus/log_cache.h

```diff
@@ -286,6 +286,13 @@ class LogCache {
     // Keeps track of uncompressed size of messages cached. This will be same as
     // log_cache_size if compression is not enabled and on secondaries
     scoped_refptr<AtomicGauge<int64_t> > log_cache_msg_size;
+
+    // Payload size of the msg that is written to the log
+    scoped_refptr<AtomicGauge<int64_t> > log_cache_payload_size;
```

Comment:
It looks like these can just be instantiated as a Counter as opposed to a Gauge because they are never reset or decremented

Reply:
Change to Counter